src/pds/your_package_name/__init__.py

```diff
@@ -1,10 +1,7 @@
 # -*- coding: utf-8 -*-
 """My PDS Module."""
-from ._version import get_versions
 
-__version__ = get_versions()["version"]
-__date__ = get_versions()["date"]
-del get_versions
+__version__ = "0.1.0"
```

Comment:
Maybe instead we should do:
```python
__version__ = pkg_resources.resource_string(__name__, "VERSION.txt").decode("utf-8").strip()
```
because then we can have in `setup.cfg`:
```ini
[metadata]
version = file: src/pds/your_package_name/VERSION.txt
```
and then just one place you need to update the version number.

Reply:
Sure, that sounds good to me. I'll update shortly.

Reply:
Updated as requested!